src/presenter/cerebral/frameworks/humbleBrowser.js

```diff
@@ -0,0 +1,6 @@
+const humbleWindow = {
+  history: () => {
+    return history;
```

Comment:
Are you referencing the global `history` deliberately? If so, I would recommend calling it out as `window.history`.